flit-server/package.json

```diff
@@ -10,7 +10,8 @@
     "license": "MIT",
     "scripts": {
         "start": "npm run serve",
-        "build": "npm run build-sass && npm run build-ts && npm run lint && npm run copy-static-assets",
+        "generate": "graphql-codegen",
+        "build": "npm run build-sass && npm run generate && npm run build-ts && npm run lint && npm run copy-static-assets",
```

Comment:
makes sense to add this into build.  I also saw references to `prestart` as a command that would run whenever `start` is issued, and various tools that try to monitor the filesystem to refresh for an easier dev workflow.  I'm not familiar enough to know when those are called for, but it seems like we'll have options if we feel limited.